include/swift/AST/AnyFunctionRef.h

```diff
@@ -151,7 +151,10 @@ class AnyFunctionRef {
     }
     llvm_unreachable("unexpected AnyFunctionRef representation");
   }
-  
+
+// Disable "only for use within the debugger" warning
+#pragma warning(push)
```

Comment:
ah I assumed if this worked with clang-cl it would work with Clang, but I was wrong - fixed